contracts/src/test/kotlin/com/r3/corda/lib/tokens/contracts/NonFungibleTokenTests.kt

```diff
@@ -143,7 +143,7 @@ class NonFungibleTokenTests : ContractTestCommon() {
                 val anotherIssuedTokenHeldByAlice = anotherIssuedToken heldBy ALICE.party
 
                 input(NonFungibleTokenContract.contractId, anotherIssuedTokenHeldByAlice)
-                output(NonFungibleTokenContract.contractId, anotherIssuedTokenHeldByAlice withNewHolder BOB.party)
+                output(NonFungibleTokenContract.contractId, anotherIssuedTokenHeldByAlice.withNewHolder(BOB.party))
```

Comment:
do we still have withNewHolder function? Or was it changed to withHolder? It's inconsistent

Reply:
There is only one function `Abstract.withNewHolder`. There is also `heldBy` which is an infix function and used when creating `AbstratToken`s.